scripts/install_tethys.bat

```diff
@@ -0,0 +1,61 @@
+# WARNING this is completely untested!!!!
```

Comment:
It's hard to say. Setting up the shell script took me a lot longer than I expected, but I kept adding features, and since shell scripting is not my forte it was a lot of Googling to try to figure out how to make it all work. Testing the script can take a while since it needs to download a lot of things, and since I was doing it on multiple platforms that made it even longer. All of that is to say I think it will be a fair about of work, but hopefully less that what has been done already.